hathor/conf/testnet.py

```diff
@@ -8,12 +8,12 @@
     # Genesis stuff
     GENESIS_OUTPUT_SCRIPT=bytes.fromhex('76a914a584cf48b161e4a49223ed220df30037ab740e0088ac'),
     GENESIS_TIMESTAMP=1577836800,
-    GENESIS_BLOCK_NONCE=3526202,
-    GENESIS_BLOCK_HASH=bytes.fromhex('000007eb968a6cdf0499e2d033faf1e163e0dc9cf41876acad4d421836972038'),
-    GENESIS_TX1_NONCE=12595,
-    GENESIS_TX1_HASH=bytes.fromhex('00025d75e44804a6a6a099f4320471c864b38d37b79b496ee26080a2a1fd5b7b'),
-    GENESIS_TX2_NONCE=21301,
-    GENESIS_TX2_HASH=bytes.fromhex('0002c187ab30d4f61c11a5dc43240bdf92dba4d19f40f1e883b0a5fdac54ef53'),
+    GENESIS_BLOCK_NONCE=826272,
+    GENESIS_BLOCK_HASH=bytes.fromhex('0000033139d08176d1051fb3a272c3610457f0c7f686afbe0afe3d37f966db85'),
+    GENESIS_TX1_NONCE=190,
+    GENESIS_TX1_HASH=bytes.fromhex('00e161a6b0bee1781ea9300680913fb76fd0fac4acab527cd9626cc1514abdc9'),
+    GENESIS_TX2_NONCE=115,
+    GENESIS_TX2_HASH=bytes.fromhex('00975897028ceb037307327c953f5e7ad4d3f42402d71bd3d11ecb63ac39f01a'),
```

Comment:
I think we should keep the genesis weights as the `MIN_WEIGHT`, otherwise we would need to create another conf just for the genesis weight. Do you see any benefits from keeping the hashes and create another conf for the genesis weights? I see only that we wouldn't need to release a new testnet.

For me we just release the `testnet-foxtrot` with the new hashes

Reply:
On the one side, we haven't change much of our full node. So, we don't a reason to release `testnet-foxtrot`. On the other side, I don't a see a problem releasing `testnet-foxtrot`.

I mean, I'm good with both choices. We just need to communicate properly to the community if we release a new version of the testnet.

Reply:
Is there a reason we need to release another testnet already? I feel like we can wait until there are larger changes that would require a new testnet.

Or is it part of the process of deploying new nodes to the mainnet?

Reply:
As far as I understood, there is no need to release another testnet. We can just update the weight of the genesis, and everything will be good.

Reply:
I changed the weight, as suggested to go back to the old weight and keep the testnet genesis unchanged. However even with the old data the testnet genesis is still wrong. I went back to the testnet genesis when we released the mainnet (the first version that we had a testnet genesis different from the mainnet) and it was wrong.

So now I see even more value to do this genesis test in all confs and I already changed the testnet name to `foxtrot`. I think we should release a new testnet.

What do you think? @msbrogli @jansegre @obiyankenobi

Reply:
I'm good with resetting it.

Reply:
OK, but let's at least make a commitment to make this the final testnet. Otherwise we will have a harder time testing upgrades (soft/hard forks) when we need.

Reply:
I agree with @jansegre that we shouldn't keep releasing new testnet without a reason. I'm wondering what genesis our current testnet is using right now, if we can't just keep it.

Reply:
As discussed in the daily today, we all agree to release a new testnet to fix the genesis.